XenAdmin/MainWindow.Designer.cs

```diff
@@ -166,6 +166,8 @@ private void InitializeComponent()
             this.deleteToolStripMenuItem = new XenAdmin.Commands.CommandToolStripMenuItem();
             this.toolStripSeparator26 = new System.Windows.Forms.ToolStripSeparator();
             this.pluginItemsPlaceHolderToolStripMenuItem2 = new System.Windows.Forms.ToolStripMenuItem();
+            this.manageCPUGroupToolStripMenuItem = new XenAdmin.Commands.CommandToolStripMenuItem();

```

Comment:
I'd rather have this under the Manage vApps button, in the same group as the latter.